src/cpp/src/prompt_lookup/prompt_lookup_impl.cpp

```diff
@@ -84,10 +84,12 @@ void ContinuousBatchingPipeline::PromptLookupImpl::step() {
         raw_perf_counters.m_new_token_times.emplace_back(main_timer.get_end_time());
     }
 
-    if (generated_len_after.empty() && 0) {
+#ifdef LOG_LEVEL_DEBUG
```

Comment:
Check `NDEBUG` instead. Example: https://github.com/openvinotoolkit/openvino.genai/blob/e312b49e6a874a4401632734554998f25b1c5265/src/cpp/src/lora_adapter.cpp#L48

Reply:
I realized the following problem only now. It doesn't make much sense to collect metrics in debug, because they are going to be different for release. `OPENVINO_LOG_LEVEL` should have been used similar to `print_compiled_model_properties()`.

Reply:
thank you, moved to use env variable as trigger